ambientweather2sqlite/database.py

```
@@ -178,140 +92,268 @@
+        self.conn.commit()
+    
+    def log_error(self, error_name: str, message: str):
+        """Log an error to the database."""
+        cursor = self.conn.cursor()
+        cursor.execute(
+            'INSERT INTO logs (error, message) VALUES (?, ?)',
+            (error_name, message)
+        )
+        self.conn.commit()
+    
+    def close(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `close`

Add return type annotation: `None`

(ANN201)